### ISSUE TITLE
Using Holo.Theme.Dialog for an activity crashes ActionBarSherlock

### ISSUE THREAD

**Issue by Ereza** [Prototik/HoloEverywhere#415]
I am using Holo.Theme.Dialog.Light (tried Holo.Theme.Dialog too) to display my activity as a dialog. This works OK on Android 4.0+, but crashes on earlier versions. The crash is the following:

```
java.lang.IllegalStateException: ActionBarView can only be used with android:layout_width="match_parent" (or fill_parent)
    at com.actionbarsherlock.internal.widget.ActionBarView.onMeasure(ActionBarView.java:814)
    at android.view.View.measure(View.java:8171)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:3132)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:245)
    at com.actionbarsherlock.internal.widget.ActionBarContainer.onMeasure(ActionBarContainer.java:181)
    at android.view.View.measure(View.java:8171)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:3132)
    at org.holoeverywhere.widget.LinearLayout.measureChildWithMargins(LinearLayout.java:214)
    at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1012)
    at android.widget.LinearLayout.measureVertical(LinearLayout.java:381)
    at android.widget.LinearLayout.onMeasure(LinearLayout.java:304)
    at android.view.View.measure(View.java:8171)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:3132)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:245)
    at android.view.View.measure(View.java:8171)
    at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:3132)
    at android.widget.FrameLayout.onMeasure(FrameLayout.java:245)
    at android.view.View.measure(View.java:8171)
    at android.view.ViewRoot.performTraversals(ViewRoot.java:801)
    at android.view.ViewRoot.handleMessage(ViewRoot.java:1727)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:123)
    at android.app.ActivityThread.main(ActivityThread.java:4627)
    at java.lang.reflect.Method.invokeNative(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:521)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:868)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:626)
    at dalvik.system.NativeStart.main(Native Method)
```

**Comment by Prototik** [comment]
Temporary solution: use themes Holo.Theme.Dialog.FixedSize and Holo.Theme.Dialog.Light.FixedSize.

**Comment by Ereza** [comment]
Thanks for the response. I just tried it and it crashes in the same way...

**Comment by Prototik** [comment]
Fixed via 536e0abee82f8b463392319d7683d7350deced48
